exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,34 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative( number ) {
+  if (number < 0 ){ 
+    return "number is negative";
+  } else {
+    return "number is not negative";
+  }
+
+}
+
+function range(number) {
+  if ( 5 < number<10) {
```

Comment:
While this is correct mathematical notation, in programming you can only compare 2 things in a single conditional statement. Therefore, this needs to be two separate conditionals, joined with an `and` symbol `number > 5 && number < 10`.